starter_code/public/javascript/APIHandler.js

```diff
@@ -1,25 +1,63 @@
 class APIHandler {
-  constructor (baseUrl) {
+  constructor(baseUrl) {
     this.BASE_URL = baseUrl;
+    const characterApi = axios.create({
+      baseURL: this.BASE_URL
+    });
   }
 
-  getFullList () {
-
+  getFullList() {
```

Comment:
You need to return this promise:

`
return   this.characterApi.get("/characters") ... ... 
`